### PR TITLE
Azure policy to create diagnostic settings

### PR DESCRIPTION
**Pull request by MattWhelan**
Includes README updates.

It's frustrating that different tools use slightly different JSON
to create Policy objects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,19 +68,33 @@ to New Relic using an [Azure Policy](https://docs.microsoft.com/en-us/azure/azur
...
-  --template-uri https://raw.githubusercontent.com/newrelic/newrelic-azure-log-ingestion/main/templates/azure-policy.json
-```
+To set up this Azure Policy, run the following commands using the Azure CLI:
 
-Replacing, as necessary, the `--resource-group` argument with the ResourceGroup
-that you created during installation.
+    az policy definition create \
+        --name New-Relic-Diagnostic \
+        --mode All \
+        --rules $( jq '.properties.policyRule' < templates/diagnostic-policy.json )
```
Comment (kolanos) [inline README.md:80]: Couldn't these be split into separate JSON files? That appears to be what Azure does in their policy examples.

Also, can these be URLs? That way the user doesn't need to clone the repo to run these commands.

File: README.md [hunk 2]
```diff
@@ -68,19 +68,33 @@ to New Relic using an [Azure Policy](https://docs.microsoft.com/en-us/azure/azur
...
+        --name New-Relic-Diagnostic-assignment \
+        --policy New-Relic-Diagnostic \
+        --assign-identity \
+        --role Contributor \
+        --identity-scope /subscriptions/${subscriptionId} \
+        --location westus2 
+        -p '{ "eventHubAuthorizationRuleId": { "value": "/subscriptions/${subscriptionId}/resourceGroups/${resource-group}/providers/Microsoft.EventHub/namespaces/${namespace-name)/authorizationRules/NewRelicLogsSharedAccessKey" } }'
+
+    az policy remediation create \
+        --name New-Relic-Diganostic-remediation \
```
Comment (kolanos) [inline README.md:93]: Typo

**Review by kolanos — CHANGES_REQUESTED** [review]
Found a typo in the README. Question about splitting rules and parameters.

File: README.md (hunk 1 above)
Comment (MattWhelan) [reply README.md]: I don't know if they can be URLs. The documentation says JSON strings or files, so I'd be surprised if that worked. 

I'll split them into separate files though.

**Review by mrickard — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kolanos) [reply README.md]: [This example](https://github.com/Azure/Community-Policy/tree/master/Policies/App%20Service/deploy-app-service-diag-log-analytics) has this in the README:

```
az policy definition create --name 'deploy-app-service-diag-log-analytics' --display-name 'Deploy App Service Diagnostics Settings' --description 'This policy deploys Diagnostic Settings for App Service to a Log Analytics workspace' --rules 'https://raw.githubusercontent.com/Azure/Community-Policy/master/Policies/App%20Services/deploy-app-service-diag-log-analytics/azurepolicy.rules.json' --params 'https://raw.githubusercontent.com/Azure/Community-Policy/master/Policies/App%20Services/deploy-app-service-diag-log-analytics/azurepolicy.parameters.json' --mode All

az policy assignment create --name <assignmentname> --scope <scope> --policy "deploy-app-service-diag-log-analytics" 

```

So it appears that URLs can be used.

Comment (MattWhelan) [reply README.md]: Alright, I've updated the README.

File: README.md [hunk 3]
```diff
@@ -68,19 +68,33 @@ to New Relic using an [Azure Policy](https://docs.microsoft.com/en-us/azure/azur
 This is the easiest option as it only needs to be setup once and all existing and future
 App Insights resources will be automatically configured to report to New Relic. Keep in
 mind there are costs associated with exporting telemetry data from Azure. You will want
-to monitor integration utilzation to ensure they meet your expectations.
+to monitor integration utilization to ensure they meet your expectations.
 
-To setup this Azure Policy, rrun the below command using the Azure CLI:
+In addition to the Azure CLI, you'll need the `jq` JSON processing tool installed.
```
Comment (kolanos) [inline README.md:73]: Believe this is no longer necessary?

**Review by kolanos — CHANGES_REQUESTED** [review]

**Review by kolanos — APPROVED** [review]

**Pull request merged**